DIP/DIP/Models/FileLogger.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DIP.Models
+{
+    public abstract class LogBase
+    {
+        public abstract void Log(string message);
+    }
+
+    public class FileLogger : LogBase
+    {
+        public string filePath = @"../DIP/Logs/Logs.txt";
+        public override void Log(string message)
+        {
+            var msg = $"{DateTime.Now.ToString()} --> {message} {Environment.NewLine}";
+            File.AppendAllText(filePath, msg);
```

Comment:
this will result to memory issues in your application, try to user a proper logging tools